app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -75,6 +76,7 @@ def build_select_button(bgi, index)
           :id       => bgi[:id] + "__" + bsi[:id],
           :type     => :button,
           :img      => img = img_value(bsi),
+          :img_url  => image_path("/assets/toolbars/" + img),
```

Comment:
this should be `image_path("toolbars/#{img}")` .. no leading `/assets/`. (And prefer interpolation over concatenation, says rubocop :))